apps/central/src/components/SchemaCreateModal.vue

```diff
@@ -137,12 +143,28 @@ export default {
       this.success = null;
       request(
         this.endpoint,
-        `mutation createSchema($name:String, $description:String, $template: String, $includeDemoData: Boolean){createSchema(name:$name, description:$description, template: $template, includeDemoData: $includeDemoData){message}}`,
```

Comment:
i prefer directly named actions, the whole mutation part of gql seems weak to me, an after thought to be complete. What i read on the interwebs is people suggest have more simple mutations instead of complex value objects

Reply:
I don't share that views your read on the internet and I am decided that I will change it eventually in the way I describe.

So the only question is: will we change it now or later? I propose later to not expand your PR too much.

Reply:
do not agree , makes the api hard to use and evolve, resolving a outcome remains the same